epimodel/tasks.py

```diff
@@ -480,6 +480,72 @@ def output(self):
         return luigi.LocalTarget(self.age_distributions)
 
 
+class AutomatedExport(luigi.Task):
```

Comment:
I think it would be better to have a switch `--automatic` or whatever for the WebExport task.
When that switch is turned on, the requires of the WebExport will not include the Gleam stuff.
Also the export will not include the gleam model data.

At the very least AutomatedExport should extend WebExport, so that it inherits the config fields from it. Right now the configs for these two tasks are duplicated.